anvil/src/eth/error.rs

```diff
@@ -150,8 +155,8 @@ pub enum InvalidTransactionError {
     #[error("intrinsic gas too low")]
     GasTooLow,
     /// returned if the transaction gas exceeds the limit
-    #[error("intrinsic gas too high")]
-    GasTooHigh,
+    #[error("intrinsic gas too high -- {}",.0.detail)]
+    GasTooHigh(ErrDetail),
```

Comment:
Hmm, i think we could have just a `String` here—if we wanna name it a bit more fancily we can just make a type like `type GasErrorDetail = String;`.